XenAdmin/Controls/CustomTreeView.cs

```diff
@@ -529,6 +532,8 @@ protected override void OnMouseDoubleClick(MouseEventArgs e)
                 Resort();
                 Refresh();
             }
+
+            DoubleClickOnRow(this, e);

```

Comment:
I think that a null check is needed here, as not all usages of CustomTreeView would have this event handler assigned.
Same for srListBox_DoubleClickOnRow() in SrPicker.cs